### ISSUE TITLE
Create a NEW Custom game list

### ISSUE THREAD

**Issue by Darknior** [gazpan/WebtroPie#31]
Maybe directly permit to users to directly creating a new custom game list from Webtropie.

**Comment by gazpan** [comment]
yes, I think clicking "options" menu, then new option "upload rom" should open a file selector.
I would also like drag and drop to work so dragging a file to the game list automatically uploads the rom.
I did think that maybe there should be something onscreen to drag to, like an item in the list with an upload icon, but it wouldn't look right at the top of a list and would be lost at the bottom of the list. So, I think for now to just do the first two options.

**Comment by Darknior** [comment]
lol you speak about this issue https://github.com/gazpan/WebtroPie/issues/33
Here i speak about adding to "option menu" a new entry to "create custom list".
- Choose a name
- Choose the skin directory
- Select games for it.

**Comment by gazpan** [comment]
oh yes, sorry, I got my wires crossed there !
